Resources/views/CRUD/base_history.html.twig

```diff
@@ -27,25 +27,27 @@ file that was distributed with this source code.
                     <table class="table" id="revisions">
                         <thead>
                         <tr>
-                            <th>{{ "td_revision"|trans({}, 'SonataAdminBundle') }}</th>
-                            <th>{{ "td_timestamp"|trans({}, 'SonataAdminBundle') }}</th>
-                            <th>{{ "td_username"|trans({}, 'SonataAdminBundle') }}</th>
-                            <th>{{ "td_action"|trans({}, 'SonataAdminBundle') }}</th>
-                            <th>{{ "td_compare"|trans({}, 'SonataAdminBundle') }}</th>
+                            <th>{{ 'td_revision'|trans({}, 'SonataAdminBundle') }}</th>
+                            <th>{{ 'td_timestamp'|trans({}, 'SonataAdminBundle') }}</th>
+                            <th>{{ 'td_username'|trans({}, 'SonataAdminBundle') }}</th>
+                            <th>{{ 'td_action'|trans({}, 'SonataAdminBundle') }}</th>
+                            <th>{{ 'td_compare'|trans({}, 'SonataAdminBundle') }}</th>
                         </tr>
                         </thead>
                         <tbody>
                         {% for revision in revisions %}
                             <tr class="{% if (currentRevision != false and revision.rev == currentRevision.rev) %}current-revision{% endif %}">
-                                <td>{{ revision.rev}}</td>
+                                <td>{{ revision.rev }}</td>
                                 <td>{% include admin.getTemplate('history_revision_timestamp') %}</td>
                                 <td>{{ revision.username}}</td>
```

Comment:
you fix ``{{ revision.rev}}`` to ``{{ revision.rev }}`` (l40)
I think you should do it here too : 
``{{ revision. username}}`` to ``{{ revision. username }}`` (l42)